src/modules/Elsa.Hosting.Management/Options/HeartbeatSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Elsa.Hosting.Management.Options;
+
+public class HeartbeatSettings
+{
+    public TimeSpan InstanceHeartbeatRhythm { get; set; } = TimeSpan.FromMinutes(1);
+    public TimeSpan InstanceDeactivatedPeriod { get; set; } = TimeSpan.FromHours(1);
```

Comment:
Although this is very descriptive, it also sounds unconventional. `HeartbeatTimeout` sounds better because we're not talking about a _periodic_ event, but a _threshold_ and it's more concise, too. Plus, it has a nice coupling with the `HeartbeatInterval` name:

- HeartbeatInterval
- HeartbeatTimeout